lib/amplitude/index.js

```diff
@@ -121,7 +124,15 @@ Amplitude.prototype.identify = function(identify){
 Amplitude.prototype.track = function(track){
   var props = track.properties();
   var event = track.event();
+  var revenue = track.revenue();
+
+  // track the event
   window.amplitude.logEvent(event, props);
+
+  // also track revenue
+  if (revenue) {
+    window.amplitude.logRevenue(revenue);
```

Comment:
On mobile, Amplitude accepts some additional data for `logRevenue` calls. Is it the same on the web?